vpr/src/route/router_lookahead_map_utils.h

```diff
@@ -294,9 +292,9 @@ typedef std::vector<std::vector<std::vector<t_reachable_wire_inf>>> t_chan_ipins
 /**
  * @brief For each tile, iterate over its OPINs and store which segment types are accessible from each OPIN
  * @param is_flat
- * @return (segments accessible on the same type, segments accessible on other layer)
+ * @return
  */
-std::pair<t_src_opin_delays, t_src_opin_inter_layer_delays> compute_router_src_opin_lookahead(bool is_flat);
+t_src_opin_delays compute_router_src_opin_lookahead(bool is_flat);
```

Comment:
Should probably just get this from the router context, but your call.

Reply:
We don't have it in the router context yet.